src/app/main.cpp

```diff
@@ -185,13 +186,13 @@ int main(int argc, char *argv[])
             setCurrentMigrationVersion();
         }
 
-        // Check if qBittorrent is already running for this user
```

Comment:
Doesn't "for this user" matter?
AFAIK, it doesn't affect qBittorrent instances of another users.

Reply:
AFAIK now it is not per user. It is per config folder.